Automattic/wp-calypso
# Updates favicons in all Jetpack Cloud

#### Changes proposed in this Pull Request

* This adds favicons for Jetpack Cloud across all environments.

#### Testing instructions



* Run the development version of Jetpack Cloud.
* Build and run the stage version of Jetpack Cloud.
* Both favicons should be changed.
* Smoke test Calypso to ensure this only affects Cloud envs.

Note: Currently, prod redirects to jetpack.com, so it's not possible to test in that environment.

Screenshots:
* Dev:<br><img width="248" alt="Screen Shot 2020-01-31 at 11 39 13 AM" src="https://user-images.githubusercontent.com/1760168/73559058-60183f00-4422-11ea-8d33-afe95d5e4082.png">
* Stage:<br><img width="249" alt="Screen Shot 2020-01-31 at 12 05 17 PM" src="https://user-images.githubusercontent.com/1760168/73559059-60183f00-4422-11ea-8d9d-70bcb36767a4.png">

Fixes 1156570014567299-as-1159371543675231.
